modules/testing/src/views/full/index.tsx

```diff
@@ -49,13 +52,43 @@ export default class Testing extends React.Component<Props, State> {
     await this.loadPreviews()
   }
 
+  resetWebchatSession = async () => {
+    // Fetches the emulator content window
+    const win: Window = document.querySelector('#bp-widget')?.['contentWindow']
+    if (win) {
+      const timeout = new Promise((_, reject) => {
+        setTimeout(() => reject('Timeout reached while waiting for the webchat to be ready'), WEBCHAT_READY_TIMEOUT)
+      })
+
+      // Wait for the webchat to be ready
+      const wait = new Promise(resolve => {
+        window.addEventListener('message', event => {
+          if (event.data && event.data.name === 'webchatReady') {
+            resolve()
+          }
+        })
+      })
+
+      // Sends a request to the emulator to create a new session (new user, new socket connection, and new conversation)
+      win.postMessage({ action: 'new-session' }, '*')
```

Comment:
This is the same logic used in the debugged to create a new session (or reset it)